torch/_inductor/ir.py

```diff
@@ -6686,6 +6689,202 @@ def codegen_collective(self, wrapper, output_name, input_names):
         )
 
 
+# TODO(yifu): replace the CollectiveKernel IR hierarchy with _CollectiveKernel.
+class _CollectiveKernel(FallbackKernel):
```

Comment:
In a subsequent PR will all of the existing CollectiveKernel stuff be removed?

Reply:
Yes